INSTALL.md

```diff
@@ -0,0 +1,236 @@
+# INSTALL / BUILD instructions for Apache Airflow
```

Comment:
We should delete this one. It's not needed once you redirected users to documentation in the `INSTALL` file